docs/how-to/dev-install.md

```diff
@@ -8,16 +8,20 @@ First clone the repository locally using [Git](https://git-scm.com/downloads). T
 
 ## Install dependencies
 
-You can choose to either develop on the host machine using a `venv` (which requires python 3.10 or later) or to run 
+You can choose to either develop on the host machine using a `venv` (which requires python 3.11 or later) or to run 
 in a container under [VSCode](https://code.visualstudio.com/)
 
 <!-- https://sphinx-design.readthedocs.io/en/latest/tabs.html# -->
 ::::{tab-set}
 
 :::{tab-item} Local virtualenv
+
+If on a DLS machine make sure you have python >3.11 running by doing `module load python/3.11`
+
 ```
-python3 -m venv venv
+python -m venv venv
```

Comment:
```suggestion
python3 -m venv venv
```

What's the behaviour on a Diamond workstation when doing `module load python/3.11; which python3`? Having just had to reinstall my laptop, `python3` is the more correct  executable to use outside of Diamond's systems: `python` doesn't exist and all programs are meant to explicitly ask for `python2 | python3`. The user is allowed to e.g. symlink `python -> python3` but there's no guarantee.